src/runtime-rs/crates/hypervisor/ch-config/src/ch_api.rs

```diff
@@ -122,3 +122,21 @@ pub async fn cloud_hypervisor_vm_fs_add(
     })
     .await?
 }
+
+pub async fn cloud_hypervisor_vm_disk_add(
```

Comment:
I dont see this function being called at all. Let me know if I missed it. I also see there is an existing function `cloud_hypervisor_vm_blockdev_add` that implements the same functionality. I think you ended up using that function later in this PR as well. Maybe this function is not required?

Reply:
Fixed.